source/AI.cpp

```diff
@@ -943,6 +947,9 @@ shared_ptr<Ship> AI::FindTarget(const Ship &ship) const
 			}
 			// Prefer to go after armed targets, especially if you're not a pirate.
 			range += 1000. * (!IsArmed(*it) * (1 + !person.Plunders()));
+			// Targets which are plundering allies earn extra scorn,
+			// especially to vengeful personalities.
+			range -= 1000 * Has(*it, gov, ShipEvent::BOARD) * (1 + person.IsVindictive());
```

Comment:
Because the "vindictive" personality confers a significant disadvantage in combat (ships get distracted attacking enemies that are already dead), no one will ever want to use "vindictive" specifically to get this effect. So, I think it'd be better to leave "vindictive" with its current meaning and not to expand it to this.